lighthouse-core/audits/network-requests.js

```diff
@@ -62,6 +63,7 @@ class NetworkRequests extends Audit {
         };
       });
 
+      // NOTE(i18n): this audit is only for debug info in the LHR and does not appear in the report.
```

Comment:
the other options are:
- actually i18n this file. Many of these column names already exist in our strings, but some (`'Status Code'`, `'MIME Type'`, etc) do not and will have to be translated. Seems not worth it?
- remove the headings and make this `debugdata`, which it really already is.  Seems like there's benefit to having it be the common table format, though, and the headings kind of serve as documentation for the columns?

Reply:
I like what we have here, continue to leave it un-i18n'd 👍